nexus/types/src/silo.rs

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Silo-related utilities and fixed data.
+
+use std::sync::LazyLock;
+
+use omicron_common::api::external::Name;
+
+/// The ID of the "default" silo.
+pub static DEFAULT_SILO_ID: uuid::Uuid =
```

Comment:
Is it weird that the `*-types` crate is gaining constants? I don't object to this, but I wonder if this is a gap in our crate organization or naming somehow.

Reply:
I guess I view this as roughly in the same ballpark as, say, the CockroachDB version constants: https://github.com/oxidecomputer/omicron/blob/dda39e3bdb6139935ec4873034c231e9d0250106/nexus/types/src/deployment/planning_input.rs#L357-L377. Slightly out of the ordinary but I don't think there's a better place to put them. (You could put them in a crate of their own but that seems excessive, since there aren't any dependencies).